src/formatDate.js

```diff
@@ -8,7 +8,61 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.find(
+    (item) => item === '/' || item === '-' || item === '.',
+  );
+
+  const separator = toFormat.find(
+    (item) => item === '/' || item === '-' || item === '.',
```

Comment:
Similar to the issue with `fromSeparator`, the logic for finding the `separator` for `toFormat` is incorrect. The `find` method is being used incorrectly here as well. Ensure that the separator is correctly identified for the `toFormat`.